n6001/hardware/ofs_n6001/build/opencl_bsp.sdc

```diff
@@ -2,10 +2,8 @@
 set_clock_groups -asynchronous  -group [get_clocks {sys_pll|iopll_0_clk_sys}] \
                                 -group [get_clocks {afu_top|pg_afu.port_gasket|user_clock|qph_user_clk|qph_user_clk_iopll|iopll_0_outclk1 \
                                                     afu_top|pg_afu.port_gasket|user_clock|qph_user_clk|qph_user_clk_iopll|iopll_0_outclk0}] \
-                                -group [get_clocks {mem_ss_top|mem_ss_*inst|mem_ss_fm*|intf_0_core_usr_clk \
-                                                    mem_ss_top|mem_ss_*inst|mem_ss_fm*|intf_1_core_usr_clk \
-                                                    mem_ss_top|mem_ss_*inst|mem_ss_fm*|intf_2_core_usr_clk \
-                                                    mem_ss_top|mem_ss_*inst|mem_ss_fm*|intf_3_core_usr_clk}]
+                                -group [get_clocks {mem_ss_top|mem_ss*inst|mem_ss_fm*|intf_0_core_usr_clk \
```

Comment:
not a real change, but attempting to increase flexibility

Reply:
we need to think about how to get rid of these clock groups eventually - the CDC bridges should handle the crossings properly, without the use of clock-groups.

Reply:
and, if the CDC bridges handled them properly, we wouldn't need to worry about the clock-name hierarchy changing and breaking everything. it should just magically happen without needing this constraint.